validator.Dockerfile

```diff
@@ -15,6 +15,9 @@ COPY go.sum go.sum
 COPY vendor/ vendor/
 
 # Copy the go source
+COPY main.go main.go
```

Comment:
Do you need these files in validator image?

Reply:
I copied the whole source code to the builder container, because the validator code imports some parts of it. `main.go` is not needed, I included it for completeness.

These files will not be in the validator image, they are only used in the builder image.